infra/terraform/platform/main.tf

```diff
@@ -57,10 +57,10 @@ module "softnas_monitoring" {
   source = "../modules/cloudwatch_alerts"
 
   name               = "${terraform.workspace}-softnas-alerts"
-  ec2_instance_ids   = "${module.user_nfs_softnas.ec2_instance_ids}"
-  ec2_instance_names = "${module.user_nfs_softnas.ec2_instance_names}"
+  ec2_instance_ids   = ["${element(module.user_nfs_softnas.ec2_instance_ids, 1)}"]
```

Comment:
So this is only passing in the second EC2 instance. 
**Maybe** worth adding a comment on top of that line to explain we're not interested in the first instance as it's not in use.

This is also fine in `dev`? I think we have one EC2 instances for SoftNAS in `dev`, isn't this going to go out of index? Maybe not, (I didn't test this by running `terraform`)

Reply:
Yes it work in dev as it will just match the first in the instance (lists are a bit weird in terraform). It is a bit hacky but the least bad solution I could come up with for now. I'll add a note to explain

Reply:
Oh OK, I wasn't sure because as you said, Terraform loops are...special 😬 and given indexes start from `0` I thought it would cause some problems when you do something like `element(..., 1)` but apparently not 🤷‍♂️ (weird!)

Reply:
Yeah so what happens if it reaches the end of the list it matches from the start again....a bit weird but handy in this case!

Reply:
ahahah